File: 1101/1101.c
// bee 1101 - Sequencia de Numeros e Soma
// Alonso Martins
// 01/05/2024
#include <stdio.h>

int max(int a, int b) { return a > b ? a : b; }
int min(int a, int b) { return a < b ? a : b; }

int soma_sequencia(int start, int end)
{
    int i, soma;
    for (soma = 0, i = start; i <= end; soma += i, i ++)
        printf("%d ", i);
    return soma;
}

int main()
{
    int a, b;
    
    while (scanf("%d %d", &a, &b) && a > 0 && b > 0)
        printf("Sum=%d\n", soma_sequencia(min(a, b), max(a, b)));
    
    return 0;
}

File: 1101/1101.java
// bee 1101 - Sequencia de Numeros e Soma
// Alonso Martins
// 29/01/2023
import java.util.Scanner;

public class Main {
    
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int M, N;
        do {
            M = in.nextInt();
            N = in.nextInt();
            if(M <= 0 || N <= 0) break;
            if(M > N) {
                int t = M; M = N; N = t;
            }
            for (int i = M; i <= N; i ++) {
                System.out.print(i+" ");
            }
            System.out.println("Sum="+((M+N)*(N-M+1)/2));
        } while(M > 0 && N > 0);
    }
}
